tests/system/requests/test_download.py

```diff
@@ -63,19 +63,20 @@
     }, {
         u'path': os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt.gz')),
         u'uncompressed':
-            os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt')),
+            os.path.realpath(os.path.join(DATA_DIR, u'gzipped.txt.gz')),
         u'content_type': PLAIN_TEXT,
         u'checksum': u'KHRs/+ZSrc/FuuR4qz/PZQ==',
         u'slices': (),
         u'metadata': {
-            u'contentEncoding': u'gzip',
+            u'Content-Encoding': u'gzip',
```

Comment:
This change is incorrect:  the key in the [Object representation](https://cloud.google.com/storage/docs/json_api/v1/objects#resource-representations) is `contentEncoding`.  By not passing it correctly, you hide the fact that the file is gzipped from the back-end, which then doesn't offer to serve it that way.  In conjunction with the change above, this change removes the whole point of testing with this file.